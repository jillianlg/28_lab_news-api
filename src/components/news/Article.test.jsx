import React from 'react';
import { render, cleanup } from '@testing-library/react';
import SingleArticle from './Article';

describe('Article component', () => {
  afterEach(() => cleanup());
  it('renders Article', () => {
    const { asFragment } = render(
      <SingleArticle
        title="tesla"
        author="kat longfellow"
        description="Elon Musk did it again..."
      />);
    expect(asFragment()).toMatchSnapshot();
  });
});

