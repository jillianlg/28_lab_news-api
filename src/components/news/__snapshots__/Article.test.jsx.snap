// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article component renders Article 1`] = `
<DocumentFragment>
  <div
    style="border: 1px solid black; padding: 8px;"
  >
    <h2>
      tesla
    </h2>
    <h4>
      kat longfellow
    </h4>
    <p
      style="font-size: 10px;"
    >
      Elon Musk did it again...
    </p>
  </div>
</DocumentFragment>
`;
